src/view/components/TabManager/index.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { connect, MapStateToProps, MapDispatchToProps, DispatchProp } from 'react-redux';
+import { Tabs as NativeTabs, Text } from 'native-base';
+
+import { Tab, TabList, ElementTab, ElementTabList, StoreTabList } from '../../../store/modules/tabs/types';
+import tabActions from '../../../store/modules/tabs/actions';
+import { State } from '../../../store/types';
+
+/** TabManager OwnProps */
+interface OwnProps {}
+
+interface StoreProps {
+  activeTab: string;
+}
+
+interface DispatchProps extends DispatchProp<State> {
+  setTabs: (tabs: ElementTabList, activeTab: ElementTab) => void;
+  setActiveTab: (activeTab: ElementTab) => void;
```

Comment:
Do you think it's the tab managers responsibility to clear tabs? Maybe the lifecycle after it is unmounted could clear the tabs so that they disappear when there is a change in page

Reply:
Yeah it is I was thinking about this on my way home and realized I had never called clear. Good catch :+1:

Reply:
I will actually make this change in my other PR if that's okay? I didn't realize I was going to have to touch TabManager again in order to actually render the tabs... but I did have to

Reply:
Fixed in my other PR